vnext/Microsoft.ReactNative/Views/VirtualTextViewManager.cpp

```diff
@@ -119,6 +119,36 @@ bool VirtualTextViewManager::UpdateProperty(
     if (IsValidColorValue(propertyValue)) {
       static_cast<VirtualTextShadowNode *>(nodeToUpdate)->m_highlightData.backgroundColor = ColorFrom(propertyValue);
     }
+  } else if (propertyName == "focusable") {
+    const auto wasHyperlink = !!span.try_as<winrt::Hyperlink>();
+    auto isHyperlink = propertyValue.Type() == React::JSValueType::Boolean && propertyValue.AsBoolean();
+    if (isHyperlink && !wasHyperlink) {
+      winrt::Hyperlink hyperlink;
+      // Underline should be handled by base class using 'textDecorationLine' prop
+      hyperlink.UnderlineStyle(winrt::UnderlineStyle::None);
```

Comment:
I'm more inclined to let components behave natively as they do in the OS, and let properties override those defaults, e.g. `textDecorationLine: 'none'`. Thoughts?